app/controllers/api/authentications_controller.rb

```diff
@@ -9,6 +9,14 @@ def edit_resource(type, id, data)
       action_result(false, err.to_s)
     end
 
+    def create_resource(_type, _id, data)
+      manager_resource, attrs = validate_auth_attrs(data)
+      task_id = AuthenticationService.create_authentication(manager_resource, attrs)
```

Comment:
would still prefer if we send in the object instead of the id,  so this would work with future CI's.